src/api/apiKey.js

```diff
@@ -1,19 +1,21 @@
-var API_KEY = 'API_KEY'
-var API_KEY_LOG_TYPE = 'ApiKeyLog'
-var ADD_API_KEY_ACTION = 'AddApiKey'
-var REMOVE_API_KEY_ACTION = 'RemoveApiKey'
-var VALIDATE_API_KEY_ACTION = 'ValidateApiKey'
+/* eslint-disable prefer-arrow-callback */
```

Comment:
why?

Reply:
There is one place where a change should be made, so let's remove this custom check disable and change it. If I understand it correctly, we should change the obfuscateApiKey_ function

Reply:
Yeah, but for some it didn't work in the arrow-function syntax. I did it this way in the beginning of the rework, don't remember much why I did it, but I remember it didn't work in the arrow-syntax for some reason. Will check it out again